RadixWalletTests/Clients/FactorSourcesClientLiveTests/FactorSourcesClientLiveTests.swift

```diff
@@ -96,7 +96,15 @@ extension FactorSource {
 					lastUsedOn: .now,
 					flags: []
 				),
-				hint: DeviceFactorSourceHint(name: name, model: "iPhone", mnemonicWordCount: .twentyFour, systemVersion: nil, hostAppVersion: nil, hostVendor: nil)
+				hint: DeviceFactorSourceHint(
```

Comment:
I think there should be a `.sampleValue` ?

Reply:
No, `DeviceFactorSourceHint` does not have sample values exposed to hosts

Reply:
We should export it